libs/types/src/system_settings.ts

```diff
@@ -109,8 +109,23 @@ export interface SystemSettings {
    * contest seat limit.
    */
   readonly bmdAllowOvervotes?: boolean;
+
+  /**
+   * The BMD print mode for the election.
+   * See {@link BmdPrintMode}.
+   */
+  readonly bmdPrintMode?: BmdPrintMode;
 }
 
+const PRINT_MODES = ['bubble_marks', 'full_ballot', 'summary'] as const;
```

Comment:
In my head, I've thought of these modes as:
- Summary ballot
- Pre-printed bubble ballot
- On-demand bubble ballot

Maybe that helps with naming the modes?

One idea might be:
- `summary_ballot`
- `bubble_ballot`
- `marks_on_preprinted_ballot`

Reply:
Thanks - yeah, I like the idea of having `bubble_ballot` refer to the full print and giving the pre-printed mode more explicit naming.
Was trying "on-demand" copy earlier and initially couldn't mentally get past that the fact that these all feel "on-demand" in some way, but coming around to it